include/pipettes/firmware/interfaces.hpp

```diff
@@ -63,22 +63,21 @@ auto get_interrupt_queues<PipetteType::THREE_EIGHTY_FOUR_CHANNEL>()
 
 namespace linear_motor {
 
-auto get_interrupt(pipette_motor_hardware::MotorHardware& hw,
+auto get_interrupt(motor_hardware::MotorHardware& hw,
                    LowThroughputInterruptQueues& queues)
     -> MotorInterruptHandlerType<linear_motor_tasks::QueueClient>;
-auto get_interrupt(pipette_motor_hardware::MotorHardware& hw,
+auto get_interrupt(motor_hardware::MotorHardware& hw,
```

Comment:
I'm changing a couple of these getter functions in this file to use `motor_hardware::MotorHardware` objects instead of `pipette_motor_hardware::MotorHardware` - this shouldn't mess with any other pipette firmware, right?

Reply:
We discussed f2f, but this will affect the L5 project so we should create separate header files to handle that.